Cakes-Co/index.html

```diff
@@ -1,16 +1,104 @@
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.cdnfonts.com/css/birthday-2" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Playfair+Display:ital@1&display=swap" rel="stylesheet">
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
 
-</html>
\ No newline at end of file
+    <header>
+      <section class="a">
+        <!-- <div> -->
+          <img
+            class="imagea"
+            src="https://images.unsplash.com/photo-1612809076670-c93d4176e065?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8OTZ8fHxlbnwwfHx8fHw%3D&auto=format&fit=crop&w=600&q=60"/>
+          <div class="flexxy">
+            <div >
+              <div class="menu"></div>
+              <div class="menu"></div>
+              <div class="menu"></div>
+            </div>
+            <div class="mylinks">
+          <a href="#home"> HOME</a>
+          <a href="#cakes"> CAKES</a>
+          <a href="#Orderin"> ORDERIN</a>
+          <a href="#home"> LESSONS</a>
+          <a href="#home"> ABOUT</a>
+        </div> 
+        <div></div>
+            <p>The best cake in town delivered to your door</p>
+          </div> 
+        </div>
+
+        <!-- <div class="mylinks">
+          <a href="#home"> HOME</a>
+          <a href="#cakes"> CAKES</a>
+          <a href="#Orderin"> ORDERIN</a>
+          <a href="#home"> LESSONS</a>
+          <a href="#home"> ABOUT</a>
+        </div>  -->
+        
+      </section>
+    </header>
+    <main>
+      <div class="b">
+        <div class="grid1">
+          <h1>Welcome</h1>
+          <p>
+          Cutter & Squidge Cakes are 100% made from dreams with 100% all natural ingredients. All the cakes are handmade daily and can be delivered within London. Our postable cakes in a tin can be delivered anywhere in mainland UK!
+          Our Red Velvet gateau is a classic recipe that joined our range of handmade cakes in 2019 and has now become one of our best-selling classic cakes! 
+           With layers of soft red velvet sponge cake and velvety smooth cream cheese frosting, this is the ultimate red velvet cake and sure to put a smile on anyone’s face!</p>
+        </div>
+        <div class="grid2">
+          <img
```

Comment:
Did you run Lighthouse? It will help you pick up accessibility issues e.g. here you should add alt text for images for accessibility